src/renderer/helpers/strings.js

```diff
@@ -70,16 +70,11 @@ export function getFirstCharacter(text, locale) {
     return ''
   }
 
-  // Firefox only received support for Intl.Segmenter support in version 125 (2024-04-16)
-  // so fallback to Array.from just in case.
-  // TODO: Remove fallback in the future
   if (Intl.Segmenter) {
```

Comment:
Now that I think about it, this if could probably be removed as well

Reply:
Uhh im not sure im following. Why can the if statement be removed?

Reply:
The if statement is checking if `Intl.Segmenter` exists so the if statement isn't necessary since it should always exist. (What's inside the if-block should stay, it's just the if-statement itself that isn't needed)